### PR TITLE
[WIP] Pro success-banner

### PR DESCRIPTION
**Pull request by craigmulligan**
Currently have only implemented the /success-banner. Still some work to do on it to confirm a/b functionality.

### CODE REVIEW

File: pages/success-banner/index.js [hunk 1]
```diff
@@ -175,17 +217,22 @@ const Footer = () =>
...
-        href="/static/success-banner.css"
-      />
-    </Head>
-    <Banner />
-    <Footer />
-  </div>;
+  <Tracker analytics={locals.analytics}>
+    <div>
+      <Head>
+        <script dangerouslySetInnerHTML={{ __html: HEAD_SCRIPT }} />
```
Comment (lurch) [inline pages/success-banner/index.js]: @Shou Do you know if this will work inside the "restricted" webview that Etcher uses?

Comment (craigmulligan) [reply pages/success-banner/index.js]: Good question @Shou do you think it will work? How would I test it locally?

Comment (Shou) [reply pages/success-banner/index.js]: Run `etcher-homepage` locally and open it in a browser. How it displays there is how it'll look in Etcher as it's completely isolated – if you want to see it in Etcher itself then you can change [the URL on line 70 in `lib/gui/index.html`](https://github.com/resin-io/etcher/blob/master/lib/gui/index.html#L70) to the localhost one and that should work fine.

**Comment by craigmulligan** [comment]
Merged this into #105

**Pull request closed without merging**